src/codegen.jl

```diff
@@ -497,6 +497,27 @@ function xcall(m::Module, name::Symbol, args...; kw...)
     xcall(Expr(:., m, QuoteNode(name)), args...; kw...)
 end
 
+"""
+    xblock(x)
+
+Create a block expression around `x`.
+"""
+xblock(x) = Expr(:block, x)
+
+"""
+    xsplat(x)
+
+Create a splatting expression around `x`, i.e. `x...`.
+"""
+xsplat(x) = Expr(:(...), x)
```

Comment:
I think same for this one, most of the cases people would just write `$(x)...`? the reason of creating these functions starts with `x` was because some of the syntax would be more readable and organized with a function.